kokoro/linux/dockerfile/test/csharp/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:stretch
```

Comment:
[Original](https://github.com/protocolbuffers/protobuf/blob/e41e2dde55feea14d8ef98c3faef140f0910c7f8/kokoro/linux/64-bit/Dockerfile#L10) was debian:latest, why change it to stretch?

Reply:
currently latest = stretch, but meaning of debian:latest can change over time, and the idea of the docker image is to provide a stable runtime environment, so pinning to a specific version seems as a logical choice.
In the past, when debian:lastest changed from jessie to streche, that broke many of our dockerfiles.

Reply:
The only difference I can see between the last successful run I can find ([22 days ago](https://source.cloud.google.com/results/invocations/9daa5926-d0ab-4a85-8716-80adaa41a100)) and this one is that the base image was updated from Debian 9.7 to 9.8 but I don't know if that's the issue (I highly doubt it is).

We could just change the script to use path variables rather than copying files. I believe it'd easier to understand and manage that way in the long run.